http/src/main/resources/init-resources/azure_vm_init_script.sh

```diff
@@ -132,6 +132,32 @@ LEONARDO_URL="${18:-dummy}"
 RUNTIME_NAME="${19:-dummy}"
 DATEACCESSED_SLEEP_SECONDS=60 # supercedes default defined in terra-azure-relay-listeners/service/src/main/resources/application.yml
 
+# Save output to file
```

Comment:
This makes sense for me to keep if:
- Maybe slightly modify to `Save output to file for debugging purposes`
- The users would not access this file via terra-ui or the terminal

Reply:
Agree on the comment!
Is there an issue with them having access to this data?
To my knowledge it should be at `/home/leoAdmin/` so users won't have access.